src/sentry/search/events/fields.py

```diff
@@ -1314,6 +1314,34 @@ def with_default(default, argument):
     return argument
 
 
+class SnQLNumericColumnNoLookup(NumericColumn):
+    measurement_aliases = {
+        MEASUREMENTS_FRAMES_SLOW_RATE,
+        MEASUREMENTS_FRAMES_FROZEN_RATE,
+        MEASUREMENTS_STALL_PERCENTAGE,
+    }
+
+    def __init__(self, name, allow_array_value=False):
+        super().__init__(name)
+        self.allow_array_value = allow_array_value
+
+    def normalize(self, value, params):
+        super().normalize(value, params)
+        return value
+
+    def get_type(self, value):
+        # `measurements.frames_frozen_rate` and `measurements.frames_slow_rate` are aliases
+        # to a percentage value, since they are expressions rather than columns, we special
+        # case them here
+        if isinstance(value, list):
+            for name in self.measurement_aliases:
+                field = FIELD_ALIASES[name]
+                expression = field.get_expression(None)
+                if expression == value:
+                    return field.result_type
```

Comment:
We currently don't support result_type on the field aliases in the QueryFields. This reference to `FIELD_ALIASES` is the legacy JSON version that has the `PseudoFields`

Reply:
Maybe we should just remove L1336-L1341 for now?

Reply:
removed!